### PR TITLE
Add spec URLs for web/api subtree

### PR DESCRIPTION
**Pull request by sideshowbarker**
Part of https://github.com/mdn/content/issues/13126

### CODE REVIEW

**Comment by sideshowbarker** [comment]
This is now just blocked on https://github.com/mdn/yari/pull/5314

**Comment by queengooborg** [comment]
It looks like the PR that was blocking this has been merged -- would you like to give this a rebase @sideshowbarker?

**Comment by sideshowbarker** [comment]
> It looks like the PR that was blocking this has been merged -- would you like to give this a rebase @sideshowbarker?

Yup, thanks for the heads-up — now rebased

File: files/en-us/web/api/aescbcparams/index.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ tags:
   - Dictionary
   - Reference
   - Web Crypto API
-spec-urls: https://www.w3.org/TR/WebCryptoAPI/#dfn-AesCbcParams
+spec-urls: https://w3c.github.io/webcrypto/#dfn-AesCbcParams
```
Comment (queengooborg) [inline files/en-us/web/api/aescbcparams/index.md:10]: I'm noticing a lot of changes to existing spec URLs in this pull request -- while the changes look good to me, would they be better in a separate PR to keep the scope of this PR concise?

File: files/en-us/web/api/css_object_model/css_declaration/index.md [hunk 2]
```diff
@@ -5,7 +5,8 @@ tags:
   - CSS
   - CSS Object Model
   - CSS Declaration
-  - Reference
```
Comment (queengooborg) [inline files/en-us/web/api/css_object_model/css_declaration/index.md:8]: I'm also noticing some tag changes, are these something we should keep separate?

**Review by queengooborg — COMMENTED** [review]
Thanks for rebasing!  Just got a couple of questions regarding some of the other changes in this PR:

File: files/en-us/web/api/aescbcparams/index.md (hunk 1 above)
Comment (sideshowbarker) [reply files/en-us/web/api/aescbcparams/index.md]: > I'm noticing a lot of changes to existing spec URLs in this pull request -- while the changes look good to me, would they be better in a separate PR to keep the scope of this PR concise?

Yup, thanks — opened https://github.com/mdn/content/pull/15982 for that

File: files/en-us/web/api/css_object_model/css_declaration/index.md (hunk 2 above)
Comment (sideshowbarker) [reply files/en-us/web/api/css_object_model/css_declaration/index.md:8]: > I'm also noticing some tag changes, are these something we should keep separate?

Yup, thanks — pulled those out

File: files/en-us/web/api/cryptokeypair/index.md [hunk 3]
```diff
@@ -7,6 +7,7 @@ tags:
   - Dictionary
   - Reference
   - Web Crypto API
+spec-urls: https://w3c.github.io/webcrypto/
```
Comment (queengooborg) [inline files/en-us/web/api/cryptokeypair/index.md:10]:
```suggestion
spec-urls: https://w3c.github.io/webcrypto/#keypair
```

File: files/en-us/web/api/document/rootelement/index.md [hunk 4]
```diff
@@ -10,6 +10,7 @@ tags:
   - Reference
   - SVG
   - root
+spec-urls: https://svgwg.org/svg2-draft/struct.html#__svg__SVGDocument__rootElement
```
Comment (queengooborg) [inline files/en-us/web/api/document/rootelement/index.md:13]:
```suggestion
browser-compat: api.Document.rootElement
```

File: files/en-us/web/api/gamepad_api/using_the_gamepad_api/index.md [hunk 5]
```diff
@@ -7,6 +7,7 @@ tags:
   - Gamepad API
   - Games
   - Guide
+spec-urls: https://w3c.github.io/gamepad/#gamepad-interface
```
Comment (queengooborg) [inline files/en-us/web/api/gamepad_api/using_the_gamepad_api/index.md:10]:
```suggestion
browser-compat: api.Gamepad
```

File: files/en-us/web/api/gamepad_api/using_the_gamepad_api/index.md [hunk 6]
```diff
@@ -330,9 +331,7 @@ if (!haveEvents) {
 
 ## Specifications
 
-| Specification                                                                | Status                       | Comment            |
-| ---------------------------------------------------------------------------- | ---------------------------- | ------------------ |
-| {{SpecName("Gamepad", "#gamepad-interface", "Gamepad")}} | {{Spec2("Gamepad")}} | Initial definition |
+{{Specifications}}
 
 ## Browser compatibility
```
Comment (queengooborg) [inline files/en-us/web/api/gamepad_api/using_the_gamepad_api/index.md:337]: Can't reach the line below this, but we should probably simplify the macro call here?

File: files/en-us/web/api/mediastream_recording_api/index.md [hunk 7]
```diff
@@ -127,9 +128,7 @@ To learn more about using the MediaStream Recording API, see [Using the MediaStr
 
 ## Specifications
 
-| Specification                                                                | Status                                       | Comment            |
-| ---------------------------------------------------------------------------- | -------------------------------------------- | ------------------ |
-| {{SpecName("MediaStream Recording", "#MediaRecorderAPI")}} | {{Spec2("MediaStream Recording")}} | Initial definition |
+{{Specifications}}
 
 ## Browser compatibility
```
Comment (queengooborg) [inline files/en-us/web/api/mediastream_recording_api/index.md:134]: Ditto for `{{Compat}}` call!

**Review by queengooborg — CHANGES_REQUESTED** [review]
LGTM, thank you!  Just got a couple of suggestions for synchronizing with BCD (feel free to defer the `{{Compat}}` macro changes to a separate PR):

File: files/en-us/web/api/gamepad_api/using_the_gamepad_api/index.md (hunk 6 above)
Comment (sideshowbarker) [reply files/en-us/web/api/gamepad_api/using_the_gamepad_api/index.md:337]: > Can't reach the line below this, but we should probably simplify the macro call here?

Yup, thanks — fixed all those

**Review by queengooborg — APPROVED** [review]
Thanks for the quick changes, LGTM!

**Pull request merged**